src/Illuminate/Session/Middleware/StartSession.php

```diff
@@ -224,7 +224,7 @@ protected function addCookieToResponse(Response $response, Session $session)
                 $this->getCookieExpirationDate(),
                 $config['path'],
                 $config['domain'],
-                $config['secure'] ?? false,
+                $config['secure'],
```

Comment:
Wouldn't it be better if it has a default value:

```
$config['secure'] ?? null,
```

Because if the config key currently does not exist, it would break existing applications. (I think it is present in almost all applications, but the default value was probably there for a reason)

Reply:
Okay, I just thought about it again, the `?? false` was there, because of the default `null` value for the `secure` key. So should be fine to be left out completely.

Reply:
Yes. Back in the days, Symfony `Cookie` didn't support `null` for secure. `null` was introduced for this "auto-secure" behaviour.